src/cli.js

```diff
@@ -39,6 +39,8 @@ export default function CLI(argv, cwd) {
     .help("h").alias("h", "help")
     .version().alias("v", "version")
     .wrap(cli.terminalWidth())
+    .strict()
+    .showHelpOnFail(false)
```

Comment:
Messing around locally, without the extra `"A command is required"` message to `demandCommand()`, I was getting less-than-useful feedback like `Unknown argument: nope`.